cmd/krew/cmd/root.go

```diff
@@ -74,9 +74,9 @@ You can invoke krew through kubectl: "kubectl krew [command]..."`,
 func Execute() {
 	if err := rootCmd.Execute(); err != nil {
 		if klog.V(1).Enabled() {
-			klog.Fatalf("%+v", err) // with stack trace
+			fmt.Fprintf(os.Stderr, "%+v\n", err) // with stack trace
```

Comment:
this probably should be still klog. users don't run in this mode anyway.

Reply:
okay, makes sense, thank you 🙏